reana_db/models.py

```diff
@@ -416,6 +417,8 @@ class Workflow(Base, Timestamp, QuotaBase):
     reana_specification = Column(JSONType)
     input_parameters = Column(JSONType)
     operational_options = Column(JSONType)
+    complexity = Column(String, default="[]")
+    workflow_priority = Column(Integer, default=0)
```

Comment:
It belongs to the `Workflow` model, so we can call it just `priority`.

Reply:
Actually, I think we don't even need a `priority` column as it can be easily computed on the fly combining complexity and overload. Same as I do in [my PR](https://github.com/reanahub/reana-db/pull/131/files#diff-9c9ecb5b02eee5df82a9c1138c56e62181e9d8d56152d45c09f7fe0a693ce980R531).

I can do it in my task if needed, together with the alembic recipe if we want to merge fast.

Reply:
You are right! Since we moved the logic to r-server from r-w-c and set priority in the message body, I guess there is no point to store it in database anymore. I've removed it and left only the complexity, since it may be useful in the future.

Reply:
>I can do it in my task if needed, together with the alembic recipe if we want to merge fast.

I was thinking exactly the same thing! Since you are also touching the database, probably it would be easier to do it once

Reply:
Ok, noted in [my PR todo list](https://github.com/reanahub/reana-db/pull/131).